airbyte-ci/connectors/pipelines/pipelines/tests/java_connectors.py

```diff
@@ -53,7 +53,7 @@ class UnitTests(GradleTask):
     """A step to run unit tests for Java connectors."""
 
     title = "Java Connector Unit Tests"
-    gradle_task_name = "test"
+    gradle_task_name = "check"
```

Comment:
The base gradle plugin defines `check` as the umbrella task for testing, `test` is java-specific. TIL.